Automattic/wp-calypso
# Signup: Remove theme upload AB test

Removes theme upload AB test that was introduced in https://github.com/Automattic/wp-calypso/pull/7683.
The actual test was removed in https://github.com/Automattic/wp-calypso/pull/12287, and the purpose of this PR is to clean up the unused code. 

# Testing instructions
1. Navigate to /start.
2. Select any design option except `Create online store`.
3. Verify that theme selection steps works correctly.
4. Verify there are no JS errors in the console.
5. Test this flow for both existing and new users.
